components/TravelMethodButtons/TravelMethodButtons.jsx

```diff
@@ -119,6 +130,22 @@ const TravelMethodButtons = () => {
         </Text>
       </Flex>
 
+      <Alert
+        status="error"
+        mb={5}
+        display={travelMethodsNumber < workOnSiteDaysNumber ? "none" : "block"}
+        width="83%"
+        borderRadius="lg"
+      >
+        <Flex>
+          <AlertIcon />
+          <AlertTitle>Travel method limit is reached</AlertTitle>
+        </Flex>
+        <AlertDescription>
+          {(answers.workMode === "wfh") ? "Please select one mode of transport for each work day" : "Please select one mode of transport for each one-site work day"}
```

Comment:
The statement 'Please select one mode of transport for each [on-site] work day' can be misleading to the respondents.
The statement could imply that the respondent _should_ select one travel method for each work day, when that isn't the case. 

For example, take the valid scenarios below:
- respondent could have selected 3 work days, and select only one travel method (implying that respondent uses the same travel method for all three work days)
- respondent could have selected 3 work days, and select 2 travel methods (implying that respondent uses one travel method to travel to work 2 days, and the other travel method to travel to work for the 3rd day)

The previous alert description would lead to less confusion imho.

Reply:
Have had a chat with @OlaShabalina about this. 
Agreed to leave the PR as-is for now, and await feedback from base team :+1: